packages/cli/src/models/local/LocalController.ts

```diff
@@ -57,13 +57,20 @@ export default class LocalController {
   }
 
   public addAll(): void {
-    const folder = Contracts.getLocalBuildDir();
-    fs.readDir(folder).forEach((file) => {
-      const path = `${folder}/${file}`;
+    const buildFolder = Contracts.getLocalBuildDir();
+    const sourceFolder = Contracts.getLocalContractsDir();
+
+    fs.readDir(buildFolder).forEach((file) => {
+      const path = `${buildFolder}/${file}`;
       if(this.hasBytecode(path)) {
         const contractData = fs.parseJson(path);
-        const contractName = contractData.contractName;
-        this.add(contractName, contractName);
+        const isLibrary = contractData.ast.nodes[1].contractKind === "library";
```

Comment:
Same as above regarding how to pick the correct ast node item and regarding the quotation.